internal/commands/root.go

```diff
@@ -162,6 +163,9 @@ func NewAstCLI(
 		accessManagementWrapper,
 	)
 	projectCmd := NewProjectCommand(applicationsWrapper, projectsWrapper, groupsWrapper, accessManagementWrapper)
+
+	importCmd := NewImportCommand(projectsWrapper, uploadsWrapper, groupsWrapper, accessManagementWrapper, byorWrapper)
```

Comment:
please check the FF (check which FF wirh Edurate team)

Reply:
Discussed at office